ReadMe.md

```diff
@@ -47,6 +47,10 @@ There's also a [Patreon](https://www.patreon.com/serenityos) if you would like t
 
 Go into the Toolchain/ directory and run the **BuildIt.sh** script. Then source the **UseIt.sh** script to put the i686-pc-serenity toolchain in your $PATH.
 
+Once you've done both of those, go into the Kernel directory, then run
+**makeall.sh**, and if nothing breaks too much, take it for a spin by using
+**run.sh**.
```

Comment:
It's 'run', not 'run.sh' and now that I say that I have no idea why it's like that..

Reply:
Oh, so it is. I must have just mentally added it, hah!